Altinn.Platform.Models/src/Profile/Models/UserProfile.cs

```diff
@@ -13,6 +14,11 @@ public class UserProfile
         /// </summary>
         public int UserId { get; set; }
 
+        /// <summary>
+        /// Gets or sets the UUID of the user
+        /// </summary>
+        public Guid? UserUUID { get; set; }
```

Comment:
In code, acronyms should be treated as words. In this case it should be `UserUuid`.

Reply:
Changed